tests/hooks_test.py

```diff
@@ -99,9 +99,8 @@ def test_python_noeval_negative():
 @pytest.mark.parametrize(
     "s",
     (
-        'log.warn("help")',
-        'logger.warn("well warn dresses found")',
-        'logging.warn("to be or not to be")',
+        'log.warn("this is depreciated")',
```

Comment:
deprecated ;)

unless you're trolling me :P

Reply:
I cow dent spill my way out of a fox.